services/file_create.proto

```diff
@@ -99,4 +100,12 @@ message FileCreateTransactionBody {
      * The memo associated with the file (UTF-8 encoding max 100 bytes)
      */
     string memo = 8;
+
+    /**
+     * An account to charge for auto-renewal of this file. If not set, or set to an
+     * account with zero hbar balance, the file's expiration must be manually extended
+     * using a FileUpdate transaction, since the network will not have authorization
+     * for any kind of auto-renewal fee collection.
+     */
+    AccountID auto_renew_account_id = 9;
```

Comment:
Existing transactions have this field named as `autoRenewAccount`. Should we keep this consistent and drop the `_id`?

Reply:
Dropped `_id`.